go/couler/conversion/argo_workflow.go

```diff
@@ -0,0 +1,62 @@
+package conversion
```

Comment:
converter?

Reply:
I was following the existing naming convention, e.g. optimization.